src/formio/components/radio.ts

```diff
@@ -0,0 +1,39 @@
+import {InputComponentSchema} from '..';
+import {OFExtensions} from '../base';
+import {ManualValues, Option, VariableValues} from '../common';
+
+type Validator = 'required';
+type TranslatableKeys = 'label' | 'description' | 'tooltip';
+
+export type RadioInputSchema = InputComponentSchema<string | null, Validator, TranslatableKeys>;
+
+/**
+ * @group Form.io components
+ * @category Base types
+ */
+interface RadioManualValuesSchema extends Omit<RadioInputSchema, 'hideLabel' | 'disabled'> {
+  type: 'radio';
+  defaultValue?: string | null;
+  multiple?: false;
+  // additional properties
+  openForms: OFExtensions<TranslatableKeys>['openForms'] & ManualValues;
+  values: Option[];
+}
+
+/**
+ * @group Form.io components
+ * @category Base types
+ */
+interface RadioVariableValuesSchema extends Omit<RadioInputSchema, 'hideLabel' | 'disabled'> {
+  type: 'radio';
+  defaultValue?: string | null;
+  multiple?: false;
```

Comment:
since this is duplicated, I think we can break this out into `interface RadioStuff` (better name to be determined :P ) and have it be extended in the two union members? That way we avoid the types accidentally diverging.